sgpc3/sgpc3.c

```diff
@@ -705,71 +705,43 @@ s16 sgpc3_get_serial_id(u64 *serial_id) {
 
 
 /**
- * sgpc3_iaq_init_continuous() - reset the SGP's internal IAQ baselines and
+ * sgpc3_tvoc_init_preheat() - reset the SGP's internal TVOC baselines and
  *                             run accelerated startup until the first
  *                             sgpc3_measure_iaq()
  *
  * Return:  STATUS_OK on success.
  */
-s16 sgpc3_iaq_init_continuous() {
+s16 sgpc3_tvoc_init_preheat() {
     return sgpc3_run_profile_by_number(PROFILE_NUMBER_IAQ_INIT_CONTINUOUS);
 }
 
 /**
- * sgpc3_iaq_init() - reset the SGP's internal IAQ baselines using the default
- *                  accelerated startup time of 64s
+ * sgpc3_tvoc_init_no_preheat() - reset the SGP's internal TVOC baselines
+ * without accelerated startup time
  *
  * Return:  STATUS_OK on success.
  */
-s16 sgpc3_iaq_init() {
-    return sgpc3_iaq_init64();
-}
-
-/**
- * sgpc3_iaq_init0() - reset the SGP's internal IAQ baselines without accelerated
- *                   startup time
- *
- * Return:  STATUS_OK on success.
- */
-s16 sgpc3_iaq_init0() {
+s16 sgpc3_tvoc_init_no_preheat() {
     return sgpc3_run_profile_by_number(PROFILE_NUMBER_IAQ_INIT0);
 }
 
 /**
- * sgpc3_iaq_init16() - reset the SGP's internal IAQ baselines using accelerated
- *                    startup time of 16s
+ * sgpc3_tvoc_init_64s_fs5() - reset the SGP's internal TVOC baselines using
+ * accelerated startup time of 64s.
  *
- * Return:  STATUS_OK on success.
- */
-s16 sgpc3_iaq_init16() {
-    return sgpc3_run_profile_by_number(PROFILE_NUMBER_IAQ_INIT16);
-}
-
-/**
- * sgpc3_iaq_init64() - reset the SGP's internal IAQ baselines using accelerated
- *                    startup time of 64s
+ * Note: legacy command for old SGPC3 fs<=5: initializes algo, for only TVOC,
+ * with preheating for 64s. Does not work for the ULP power mode!
```

Comment:
I would formulate it as `does not work in conjunction with sgpc3_set_power_mode` ULP is never used as term..

Reply:
Hmm in the `sgpc3_set_power_mode` documentation we state:
```
 * @power_mode: Power mode to set:
 *              0: Ultra low power mode (30s measurement interval)
 *              1: (Default) Low power mode (2s measurement interval)
```

We could just add something like ULP (ultra low power) and LP (low power) there.

Reply:
yes that'd work, but IMHO it's easier to match against the function if someone is not sure..

Reply:
Fixed: See https://github.com/Sensirion/embedded-sgp/pull/51/commits/4a3818e87d2c86efe5f21ebee01a41329a01be43.